docs/Radio-hardware-specifications-for-EdgeTX.md

```diff
@@ -51,16 +65,19 @@
 * I2C EEPROM
 * Bluetooth module via UART
 * IMU (e.g. LSM6DS33)
+* I2C port expander
+  * if port expanders are used for inputs, the interrupt output pin(s) of the port expander(s) shall be connected to an interrupt capable input pin of the MCU
 * SpaceMouse module
 
 ---
 
 ## Black-and-white radios
 
 ### Mandatory hardware features (B/W)
-* MCU: STM32F2 and STM32F4 with minimally 512 kByte flash (for new radios only STM32F4 with minimally 1 MB flash)
-  <br/> STM32F2 running at 120 MHz, STM32F4 running at 168 MHz
+* MCU: STM32F4 with minimally 1 MB flash running at 168 MHz
```

Comment:
```suggestion
* MCU: As of EdgeTX 2.10, STM32F4 with minimally 1 MB flash running at 168 MHz. From EdgeTX v2.11 onwards, STM32H562/563 will also be supported. STM32F based designs will not be accepted as of EdgeTX v3.0.
```

Reply:
I suggest to explicitly add "New" to the last sentence, thus:
`New STM32F based designs will not be accepted as of EdgeTX v3.0.`